wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,164 @@
+
+This can be done by entering this to the Command Line Interface (CLI): 
+```prolog
+git add .
+git commit -m "<explain your changes here>"
+```
+However, some editors (ie. Visual Studio Code) may provide a Git interface.
+
+After committing your changes, you need to push the commits to GitHub so everyone can see it. To do this, enter the following to your Command Line Interface:
+
+```prolog
+git push origin <branch>
+```
+This should prompt you to enter your GitHub login credentials. After you enter them, this should push your commits to GitHub.
+
+_Note: However, If you're authentication details were saved in Git already, no prompt will appear and will proceed directly to push._
+
+## Pull Requests
+
+After pushing your commits, you must make a "Pull Request". This lets us know that your commits are ready to be reviewed.
+
+
+Go to the "Pull Requests" tab.
+
+![Pull Request Tab](6.JPG "Pull Request Tab")
+
+This would lead you to the pull requests page.
+
+![PR UI](11.jpg "Pull Requests Page")
+
+This is the Pull Request Page. Now click on "New pull request"
+
+![Compare Changes Page](12.jpg "Comparing Changes")
+
+We want to compare ``ppy:master`` against your branch.
+
+From here, select the branch that contains your commits besides your "head fork".
+
+![Comparing Changes 2](13.jpg "Comparing Changes 2")
+
+_However, if you made changes in your master branch, the changes will be shown without the need to change anything in the head fork options._
+
+Now after viewing your changes, click "Create Pull Request" and fill it with the following details:
+
+- **Title** : the summary of the changes you have made
+  - Note: it is really important to have a meaningful and clear title, otherwise it will cause confusion
+
+- **Leave a Comment** : the details of the changes you have made
+
+![PR Creation Dialogue](14.jpg "PR Creation Dialogue")
+
+When you are finished, click on the "Create pull request" button and wait for a reviewer to check and, possibly, merge your changes.
+
+**Your commit messages must thoroughly explain your changes, otherwise, they will not get merged!**
+
+## FAQ
+
+###  Why is the old osu!wiki not staying?
+
+Due to abysmal performance, the old wiki is getting deprecated
+as stated [here](https://github.com/ppy/osu-wiki/issues/69).
+
+###  How would this differ from the old wiki?
+
+- We use GitHub for collaboration. You can make mass changes easily and efficiently and you can delay your changes as long as you like.
+- Unlike the old osu!wiki, changes has to be made via pull requests which are then reviewed by staff or someone with write permissions. This adds a layer of security against spammers and vandals.
+- Linking is done manually, and must be done using static linking.
+- **No Templates**. You must duplicate the template and paste it accordingly to the article you are editing.
+- Deletion of media is easier and requires no special wiki function. However, this has to go under go the pull request protocol.
```

Comment:
go under go?

Adding in about add and rename: `Addition, deletion, and renaming of media is easier and requires no special wiki function. However, this has to go under the pull request protocol.`

Keeping the history is another problem tho.